src/main/java/fr/techad/edc/client/io/EdcReader.java

```diff
@@ -45,4 +46,14 @@
      * @throws InvalidUrlException if the url is malformed
      */
     Map<String, Map<String, String>> readLabels(Set<String> languageCodes) throws IOException, InvalidUrlException;
+
+    /**
+     * Read the translated popover label for the given language code
+     *
+     * @param languageCode
+     * @return an object
+     * @throws IOException
+     * @throws InvalidUrlException
+     */
+    I18NContent readLabel(String languageCode) throws IOException, InvalidUrlException;
```

Comment:
This method should replace **Map<String, Map<String, String>> readLabels(Set<String> languageCodes)**: 
 I18NContent readLabel(Set<String> languageCodes)